src/scripts/eos-analysis

```diff
@@ -397,6 +397,10 @@ Runs a list of subcommands.
         help = 'The base directory for the storage of data files. Can also be set via the EOS_BASE_DIRECTORY environment variable.',
         dest = 'base_directory', action = 'store', default = get_from_env('EOS_BASE_DIRECTORY', './')
     )
+    parser_run.add_argument('-d', '--dry-run',
+        help = 'Whether or not to perform a dry run of the analysis.',
+        dest = 'dry_run', action = 'store_true', default = False
```

Comment:
I wonder if we should invert this argument to force the user to check its analysis before it is ran.

Reply:
I was copying what is done in other programms, e.g., ``make``. The ``--dry-run`` or similar arguments are similar to what I implemented, and not inverted. In the end, we have to trust the users.